frontend/src/metabase/lib/settings.ts

```diff
@@ -150,34 +196,58 @@ class MetabaseSettings {
     );
   }
 
+  /**
+   * @deprecated use getSetting(state, "enable-password-login")
+   */
   isPasswordLoginEnabled() {
     return this.get("enable-password-login");
   }
 
+  /**
+   * @deprecated use getSetting(state, "search-typeahead-enabled")
+   */
   searchTypeaheadEnabled() {
     return this.get("search-typeahead-enabled");
   }
 
+  /**
+   * @deprecated use getSetting(state, "anon-tracking-enabled")
+   */
   trackingEnabled() {
     return this.get("anon-tracking-enabled") || false;
   }
 
+  /**
+   * @deprecated use getSetting(state, "anon-tracking-enabled")
+   */
```

Comment:
This comment still mentions `anon-tracking-enabled` but from a text search `uploadsEnabled` isn't seemed to be used anywhere. The only file that contains the string `uploadsEnabled` is `frontend/src/metabase/collections/containers/CollectionContent.jsx` which defines its own version.

Can you double-check that maybe we could remove this `uploadsEnabled` function altogether?